pkgs/top-level/all-packages.nix

```diff
@@ -11810,6 +11810,7 @@ with pkgs;
   etcd = etcd_3_5;
   etcd_3_4 = callPackage ../servers/etcd/3.4.nix { };
   etcd_3_5 = callPackage ../servers/etcd/3.5 { };
+  etcd_3_6 = callPackage ../servers/etcd/3_6 { };
```

Comment:
In nixpkgs, version directories should use "_" instead of ".". I'll make another PR normalizing name format after etcd current PRs are merged.